webhook_server/tests/test_github_api.py

```
@@ -1397,7 +1401,13 @@ async def test_clone_repository_empty_checkout_ref(
             # Setup mocks
             mock_config = Mock()
             mock_config.repository_data = {"enabled": True}
-            mock_config.get_value.return_value = None
+
+            def get_value_side_effect(value: str, *_args: Any, **_kwargs: Any) -> Any:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `*_args`

(ANN401)